web/src/main/bvmui/src/router.js

```diff
@@ -18,7 +18,7 @@ const router = new Router({
         {
             path: '/',
             name: 'home',
-            redirect: "/login"
+            redirect: '/bookies'
```

Comment:
Are you sure?
Isn't the  first page the 'login' page?

Reply:
Before each route it checks if it's logged in. So when you go to / you go too bookies if you are logged otherwise it redirects to /login